history-network.md

```diff
@@ -222,73 +211,55 @@ content               = SSZ.serialize(epoch_accumulator)
 content_key           = selector + SSZ.serialize(epoch_accumulator_key)
 ```
 
-#### Encoding Content Values for Validation
-
-The encoding choices generally favor easy verification of the data, minimizing decoding. For
-example:
-- `keccak(encoded-uncles) == header.uncles_hash`
-- Each `encoded-transaction` can be inserted into a trie to compare to the
-  `header.transactions_root`
-- Each `encoded-receipt` can be inserted into a trie to compare to the `header.receipts_root`
-
-Combining all of the block body in RLP, in contrast, would require that a validator loop through
-each receipt/transaction and re-rlp-encode it, but only if it is a legacy transaction.
-
-#### Content ID
-
-We derive a `content-id` from the `content_key` as `H(content_key)` where `H` denotes the SHA-256 hash function, which outputs 32-byte values. The `content-id` represents the key in the DHT that we use for `distance` calculations.
-
-
-### Radius
-
-We define a `distance` function that maps a `node-id` and `content-id` pair to a 256-bit unsigned integer identically to the `distance` function for pairs of `node-id` values.
-
-Each node specifies a `radius` value, a 256-bit unsigned integer that represents the data that a node is "interested" in.
 
-```
-interested(node, content) = distance(node.id, content.id) <= node.radius
-```
+### Algorithms
 
-A node is expected to maintain `radius` information for each node in its local node table. A node's `radius` value may fluctuate as the contents of its local key-value store change.
+#### The "Header Accumulator"
 
+The "Header Accumulator" is based on the [double-batched merkle log accumulator](https://ethresear.ch/t/double-batched-merkle-log-accumulator/571) that is currently used in the beacon chain.  This data structure is designed to allow nodes in the network to "forget" the deeper history of the chain, while still being able to reliably receive historical headers with a proof that the received header is indeed from the canonical chain (as opposed to an uncle mined at the same block height).  This data structure is only used for pre-merge blocks.
```

Comment:
Perhaps it is useful to indicate the status of post-merge proofs:

```suggestion
The "Header Accumulator" is based on the [double-batched merkle log accumulator](https://ethresear.ch/t/double-batched-merkle-log-accumulator/571) that is currently used in the beacon chain.  This data structure is designed to allow nodes in the network to "forget" the deeper history of the chain, while still being able to reliably receive historical headers with a proof that the received header is indeed from the canonical chain (as opposed to an uncle mined at the same block height).  This data structure is only used for pre-merge blocks. The system for proving headers of execution block headers post-merge is in the design phase and will be outlined in this document at a later stage.
```

Reply:
Going to leave this out for now.